build/Targets/Packages.props

```diff
@@ -80,7 +80,7 @@
     <MicrosoftVisualStudioImagingInterop140DesignTimeVersion>14.3.25407</MicrosoftVisualStudioImagingInterop140DesignTimeVersion>
     <MicrosoftVisualStudioInteractiveWindowVersion>2.0.0-rc3-61304-01</MicrosoftVisualStudioInteractiveWindowVersion>
     <MicrosoftVisualStudioLanguageCallHierarchyVersion>15.0.26201-alpha</MicrosoftVisualStudioLanguageCallHierarchyVersion>
-    <MicrosoftVisualStudioLanguageIntellisenseVersion>15.0.26201-alpha</MicrosoftVisualStudioLanguageIntellisenseVersion>
+    <MicrosoftVisualStudioLanguageIntellisenseVersion>15.0.26509-alpha</MicrosoftVisualStudioLanguageIntellisenseVersion>
```

Comment:
We should really be moving our entire VS SDK as a single item.  Otherwise we are just cobbling bits together and hoping that they work.  It's risky and doesn't represent our actual shipping environment.